reqs/management/commands/import_reqs.py

```diff
@@ -65,8 +76,8 @@ def __init__(self):
     def from_row(self, row):
         """Retrieve/create/update a Policy object"""
         policy_number = int(row['policyNumber'])
-        uri_key = find_key(("uriPolicyID", "uriPolicyId"), row)
-        omb_key = find_key(("ombPolicyID", "ombPolicyId"), row)
+        uri_key = find_key(FIELDS["uri_policy_ids"], row)
+        omb_key = find_key(FIELDS["omb_policy_ids"], row)
```

Comment:
Small detail, but would it make sense for `find_key` to lookup the values in `FIELDS` instead of calculating the field names in the caller?